content/derisking-government-tech/home/buying-development-services.md

```diff
@@ -50,7 +50,7 @@ Done well, market research shapes the final product as well as a product’s req
 
 Consult our detailed guidance for [conducting market research]({{ "/derisking-government-tech/resources/market-research/" | url }}).
 
-### Writing the solicitation using 18F’s Agile Contract Format {#writing-the-solicitation-using-18f’s-agile-contract-format}
+### Writing the solicitation using 18F’s Agile Contract Format {#writing-the-solicitation-using-18fs-agile-contract-format}
```

Comment:
clarification: I noticed that you removed most of the hardcoded ID links, but not this one. Is there a reason for that? Perhaps something apostrophe-related?

Reply:
I've manually changed some of the heading links to remove some non-word characters (smart quotes / apostrophes, parentheses) to keep those characters out of hyperlinks used elsewhere in the guide source code, for readability and to remove the possibility of confusion about character escaping. That's not necessary and if it seems unhelpful from a maintainability standpoint, I can restore the defaults as a follow-up.

Reply:
The reason I ask is because that should automatically be happening in the heading links function — it should "slugify" the string and do that removal for you.

The fact that that's not happening is concerning, and is a larger thing to look into. I don't have any concerns about going the manual route, but I'd suggest open an new issue that heading links aren't getting slugified properly.